x-pack/plugins/graph/public/components/search_bar.test.tsx

```diff
@@ -58,6 +58,9 @@ function getServiceMocks() {
       query: {
         savedQueries: {},
       },
+      indexPatterns: {
+        find: jest.fn(),
```

Comment:
Hm, why has it started calling `indexPatterns.find`?

Reply:
The `QueryStringInput` used to reference the `indexPatterns` service already

https://github.com/elastic/kibana/blob/26a4fe881b477ba873d3ac33b126c0019dc1d5bd/src/plugins/unified_search/public/query_string_input/query_string_input.tsx#L196

And within the `fetch*` function call `find`:
https://github.com/elastic/kibana/blob/26a4fe881b477ba873d3ac33b126c0019dc1d5bd/src/plugins/unified_search/public/query_string_input/fetch_index_patterns.ts#L22

So in theory this PR shouldn't have changed anything in here.